fix: Minor edits in simplified_transaction_flow.adoc (#944)
starknet-io/starknet-docs: components/Starknet/modules/architecture_and_concepts/pages/Accounts/simplified_transaction_flow.adoc

The key stages of transaction lifetime are:

> . The sequencer selects a transaction from the mempool and calls the `__validate__` function.

Author: ```suggestion
. The sequencer selects a transaction from the mempool and calls the `+__validate__+` function.
```

> . If the transaction iss valid, the sequencer calls the `__execute__` function.

Author: ```suggestion
. If the transaction is valid, the sequencer calls the `+__execute__+` function.
```

> . If `__execute__` runs successfully, the sequencer includes the transaction in the block, charges the fee and proceeds to work on the next transaction.

Author: ```suggestion
. If `+__execute__+` runs successfully, the sequencer includes the transaction in the block, charges the fee, and proceeds to work on the next transaction.
```

. After completing the block, the sequencer sends the block to the prover.

// Why is this section relevant to this topic?

[id="a_payment_mechanism"]
== The payment mechanism

The sequencer receives fees in ETH in return for including transactions in a block.

> For more details on how the transaction fee is computed, see xref:../Network_Architecture/fee-mechanism.adoc[the fees section]

Author: ```suggestion
For more details on how the transaction fee is computed, see xref:../Network_Architecture/fee-mechanism.adoc[Gas and transaction fees].
```
